README.md

```diff
@@ -693,8 +693,8 @@ username = "your-email"
 password = "password-or-api-key-for-confluence-cloud"
 # If you are using Confluence Cloud add the /wiki suffix to base_url
 base-url = "http://confluence.local"
-h1-title = true
-h1-drop = true
+h1_title = true
+h1_drop = true
```

Comment:
```suggestion
drop-h1 = true
```

Reply:
I don't know if it really matters but your suggestion is visually troublesome for me.  Changing the ordering, using hyphens and underscores intermixed, is inconsistent. The solution I offered works, and follows a pattern. 
Make the user experience better by following patterns that make the features intuitive.